src/rviz/InteractiveMarkerViewer.cpp

```diff
@@ -49,6 +50,42 @@ FrameMarkerPtr InteractiveMarkerViewer::addFrame(
   return marker;
 }
 
+TSRMarkerPtr InteractiveMarkerViewer::visualizeTSR(
+  aikido::constraint::TSR const &_tsr,
+  int nSamples,
+  std::string basename)
+{
+  using dart::dynamics::Frame;
+  using dart::dynamics::SimpleFrame;
+  using aikido::constraint::TSR;
+
+  auto sampler = _tsr.createSampleGenerator();
+  auto state = _tsr.getSE3()->createState();
+  auto tsrMarker = std::make_shared<TSRMarker>();
+
+  if (basename == "")
+  {
+      std::ostringstream ost;
+      ost << &tsrMarker;
+      basename = ost.str();
+  }
+  for(int i = 0; i < nSamples && sampler->canSample(); ++i)
+  {
+    assert(sampler->sample(state));
+
+    std::stringstream ss;
+    ss << "Marker[" << basename << "].frame[" << i << "]";
```

Comment:
Nit: `Marker` -> `TSRMarker` to avoid collisions with other markers we add in the future.